File: java/Samples/BiometricStandards/CS/FIRecordSampleCS/AddFingerForm.cs
using System;
using System.Windows.Forms;
using Neurotec.Biometrics.Standards;

namespace Neurotec.Samples
{
	public partial class AddFingerForm:Form
	{
		public AddFingerForm()
		{
			InitializeComponent();

			cbFingerPosition.Items.AddRange(Enum.GetNames(typeof(BdifFPPosition)));
			cbFingerPosition.SelectedIndex = 0;
		}

		public BdifFPPosition FingerPosition
		{
			get
			{
				return (BdifFPPosition)Enum.Parse(typeof(BdifFPPosition), cbFingerPosition.SelectedItem.ToString());
			}
			set { cbFingerPosition.SelectedText = value.ToString(); }
		}
	}
}

File: java/Samples/BiometricStandards/VB/FIRecordSampleVB/AddFingerForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
Imports Neurotec.Biometrics.Standards

Partial Public Class AddFingerForm
	Inherits Form
	Public Sub New()
		InitializeComponent()

		cbFingerPosition.Items.AddRange(System.Enum.GetNames(GetType(BdifFPPosition)))
		cbFingerPosition.SelectedIndex = 0
	End Sub

	Public Property FingerPosition() As BdifFPPosition
		Get
			Return CType(System.Enum.Parse(GetType(BdifFPPosition), cbFingerPosition.SelectedItem.ToString()), BdifFPPosition)
		End Get
		Set(ByVal value As BdifFPPosition)
			cbFingerPosition.SelectedText = value.ToString()
		End Set
	End Property
End Class
